multimedia/templates/multimedia/base.html

```diff
@@ -0,0 +1 @@
+{% extends "base.html" %}
```

Comment:
Actually, I would like to propose another way: I think all the digital content we "sell" here can actually have a single, unified UI even when showing the details page. I mean, even Google Playstore can use the same layout to display all its different products. (Not sure about Apple AppStore) So we don't need separate apps/templates for each type of multimedia